src/network-config.js

```diff
@@ -1,26 +1,31 @@
 import {
-  getLocalChainId,
+  getDefaultEthNode,
   getEnsRegistryAddress,
   getFortmaticApiKey,
+  getLocalChainId,
   getPortisDappId,
 } from './local-settings'
 
-const localEnsRegistryAddress = getEnsRegistryAddress()
 const fortmaticApiKey = getFortmaticApiKey()
 const portisDappId = getPortisDappId()
 
-export const networkConfigs = {
-  main: {
+const providedDefaultEthNode = getDefaultEthNode()
+const providedEnsRegistryAddress = getEnsRegistryAddress()
+
+const networkConfigurations = {
+  // Mainnet
+  mainnet: {
     addresses: {
       ensRegistry:
-        localEnsRegistryAddress || '0x00000000000C2E074eC69A0dFb2997BA6C7d2e1e',
+        providedEnsRegistryAddress ||
+        '0x00000000000C2E074eC69A0dFb2997BA6C7d2e1e',
     },
-    nodes: {
-      defaultEth: 'wss://mainnet.eth.aragon.network/ws',
+    endpoints: {
+      read: providedDefaultEthNode || 'wss://mainnet.eth.aragon.network/ws',
```

Comment:
Much prefer this naming! ✨ wonder if `eth` is more suitable than `read`?

Reply:
AFAIK read's used as we separate our "read" provider for fetching info, from our write provider (the wallet provider). I actually think this is more clear than `eth`. 😃